doc/getting-started.md

```diff
@@ -106,6 +106,43 @@ Using the *shared storage* strategy, you can simulate *dedicated storage* by
 on.
 
 
+Serial 
+------
+
+Many `AArch64` devices do not come with a display or can be used in headless mode (as an example - a small home server which doesn't need a display).
+
+In those cases, using serial to interact with the device can be required. Especially before your operating system of choice is loaded.
+
+Tow-Boot uses the same baud rate for all platforms and devices: `115200`. This applies to all platforms, even those which generally default to another serial baud rate.
+
+You can use the serial console of your preference - `screen` , `picocom` , `minicom` and others.
+
+### Examples for connecting with serial
+
+Examples to connect to the device while installing or booting Tow-Boot from another device:
+
+```shell-session
+$ screen /dev/ttyUSB0 115200
+...
+Press Ctrl+a and K to exit
+```
+
+```shell-session
+$ picocom /dev/ttyUSB0 -b 115200
+...
+Press Ctrl+a and Ctrl+x to exit
+```
+
+```shell-session
+$ minicom -D /dev/ttyUSB0 -b 115200
+...
+
```

Comment:
Is this blank line really useful here ?

Reply:
I've completely missed that one! Thank you
Will change it in a few